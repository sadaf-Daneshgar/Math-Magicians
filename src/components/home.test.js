import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home';

test('Home component snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
