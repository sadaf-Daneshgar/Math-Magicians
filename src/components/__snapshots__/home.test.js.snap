// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot 1`] = `
<div
  className="home-cont"
>
  <h2>
    Weclome to our page!
  </h2>
  <p
    className="para"
  >
    If sufficiently systematic numerical observations or trappings of birds or other animals on migration are made, and the accumulating totals plotted on probability paper on a time base or abscissa, they will frequently plot to a fairly good straight line. This means that the “distribution” of the observations in time is approximately Gaussian, and from the plot the peak of migration and the standard deviation of the timing can be ascertained with some accuracy. This can be done, for most passage—migrant species, for both spring and fall migrations, and so the distance or interval between peaks can be ascertained also.This can be done, for most passage—migrant species, for both spring and , and so the distance or interval between also.
    <br />
    <br />
    What is migration in maths? A mathematical model of migration may be defined as a simplified description of real migration processes, where all the crucial links between the real “participants” of spatial movements migrants and factors of migration are expressed mathematically (see Fundamentals of Mathematical Modeling for Complex Systems). What is migration in maths? A mathematical model of migration may be defined as a simplified description of real migration processes, where all the crucial links between the real “participants” of spatial movements migrants and factors of migration are expressed mathematically (see Fundamentals of Mathematical Modeling for Complex Systems Modeling for Complex Systems).
  </p>
</div>
`;
